pages/LoginView.neu.page.ts

```diff
@@ -58,6 +58,12 @@ export class LoginViewPage {
     await setPasswordButton.click();
     return newPassword;
   }
+  // Erster Login mit Passwortänderung
+  public async firstLogin(username: string, password: string) : Promise<StartViewPage> {
```

Comment:
Dieser Name beschreibt besser, was die Methode macht:
```suggestion
  public async loginAndUpdatePassword(username: string, password: string) : Promise<StartViewPage> {
```